plugin.xml

```diff
@@ -32,7 +33,7 @@
                 <source url="https://github.com/CocoaPods/Specs.git"/>
             </config>
             <pods use-frameworks="true">
-                <pod name="PurchasesHybridCommon" spec="1.10.0"/>
+                <pod name="PurchasesHybridCommon" spec="3.2.0"/>
```

Comment:
note: cordova for Android will be broken until the android side of this is done too